pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -124,8 +123,8 @@ async def maybe_ack(ack_id: str) -> None:
                         asyncio.ensure_future(maybe_ack(ack_id))
                     ack_ids = []
 
-                log.warning(
-                    'Ack request failed, better luck next batch', exc_info=e)
+                log.warning('ack request failed, better luck next batch',
```

Comment:
Do you think `better luck next batch` is really necessary? 😂 Perhaps more importantly, it is difficult to infer the meaning of it just from the wording (e.g. are messages dropped or not?). Looks like it is the result of a retryable error, maybe we can make this more clear with a message such as `ack request failed and will be retried` ?

Reply:
Actually I guess it is difficult to know the behavior, and it is not the same request that will be retried by the looks of it - depends on how many messages are in the batch. So my wording above also sounds misleading. Maybe fine to leave.

Reply:
Yeah, for now I'll leave it since I don't quite understand all the circumstances where wed retry / not / etc, but agreed this could get further improved.